Dockerfile.service_discovery

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+ARG RUST_VERSION=1.65
+FROM rust:${RUST_VERSION} AS builder
+
+# Dockerfile for building Eclipse Chariott runtime container
```

Comment:
nit: Consider changing this to say:
Dockerfile for building Eclipse Chariott Service Discovery container